src/VisualStudio/IntegrationTest/IntegrationTests/VisualBasic/BasicChangeSignatureDialog.cs

```diff
@@ -14,6 +16,8 @@ public class BasicChangeSignatureDialog : AbstractEditorTest
     {
         protected override string LanguageName => LanguageNames.VisualBasic;
 
+        private const string ChangeSignatureDialogAutomationId = "ChangeSignatureDialog";
```

Comment:
Have we came to a conclusion of whether or not we should be sharing constants like this with the actual product binaries? This seems kinda fragile.